bfd/src/main/resources/application.bfd.properties

```diff
@@ -1,11 +1,15 @@
 bfd.keystore.password=changeit
 bfd.keystore.location=/bb.keystore
 bfd.serverBaseUrl=https://fhir.backend.bluebutton.hhsdevcloud.us/v1/fhir/
+bfd.hicn.hash=https://bluebutton.cms.gov/resources/identifier/hicn-hash
 bfd.connectionTimeout=5000
 bfd.socketTimeout=5000
 bfd.requestTimeout=5000
 bfd.pagesize=100
 
+bfd.hash.pepper=6E6F747468657265616C706570706572
+bfd.hash.iter=2
```

Comment:
These two should refer to the environment variables -- AB2D_HICN_HASH_PEPPER & AB2D_HICN_HASH_ITER -- with a fallback to the defaults listed here. (I assume these work with BFD Sandbox).

Reply:
Okay. Sounds good. Yes, those work with the BFD sandbox although I think they recommend different values.